pipelines/skylab/optimus/Optimus.changelog.md

```diff
@@ -1,3 +1,10 @@
+# 4.1.9
+
+2020-12-7 (Date of Last Commit)
```

Comment:
I don't think this should include any changes to optimus or SS2. The post processing pipeline is not a versioned pipeline

Reply:
agree and removed